ister_gui.py

```diff
@@ -1110,6 +1114,11 @@ def update_widgets(self):
         if not dns_res:
             dns_res = 'none found'
 
+        if 'none found' in ''.join([interface_res, mask_res, ip_res, dns_res]):
```

Comment:
The text 'none found' seems to conflict with the set_text message. Is there a way to get them better aligned or maybe just explain away my confusion.